collection-scripts/gather_metallb_logs

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+METALLB_NS=($(oc get subs -A --field-selector=metadata.name=metallb-operator -o jsonpath='{.items[*].metadata.namespace}'))
+METALLB_PODS_PATH="${BASE_COLLECTION_PATH}/namespaces/${METALLB_NS}/pods"
+
+if [ -z "${METALLB_NS}" ]; then
+    echo "INFO: MetalLB not detected. Skipping."
+    exit 0
+fi
+
+function gather_frr_logs() {
+    declare -a FILES_TO_GATHER=("frr.conf" "frr.log" "daemons" "vtysh.conf")
+    declare -a COMMANDS=("show running-config" "show bgp ipv4" "show bgp ipv6" "show bgp neighbor" "show bfd peer")
+    LOGS_DIR="${METALLB_PODS_PATH}/${1}/frr/frr/logs"
+
+    for FILE in "${FILES_TO_GATHER[@]}"; do
+        oc -n ${METALLB_NS} exec ${1} -c frr -- sh -c "cat /etc/frr/${FILE}" > ${LOGS_DIR}/${FILE}
+    done
+
+    for COMMAND in "${COMMANDS[@]}"; do
+        echo "###### ${COMMAND}" >> ${LOGS_DIR}/dump_frr
+        echo "$( oc -n ${METALLB_NS} exec ${1} -c frr -- vtysh -c "${COMMAND}")" >> ${LOGS_DIR}/dump_frr
+    done
+}
+
+oc adm inspect --dest-dir must-gather "ns/${METALLB_NS}"
+
+SPEAKER_PODS="${@:-$(oc -n ${METALLB_NS} get pods -l component=speaker -o jsonpath='{.items[*].metadata.name}')}"
+
+for SPEAKER_POD in ${SPEAKER_PODS[@]}; do
```

Comment:
How many pods is this? Should this be threaded?

Reply:
Those pods deployed from a daemonset, so as many nodes as the cluster has.

Reply:
I'll add. thank you.